src/main/java/seedu/duke/parsermanager/ParserManager.java

```diff
@@ -142,12 +142,15 @@ private Parser parseListCommand(String commandDetail) throws UndefinedSubCommand
         boolean isListProperty = listCommandTypeAndFlags.get(0).trim().equals(PROPERTY_FLAG);
         boolean isListClient = listCommandTypeAndFlags.get(0).equals(CLIENT_FLAG);
         boolean isListEverything = listCommandTypeAndFlags.get(0).equals(EVERYTHING_FLAG);
+        boolean isListPairs = listCommandTypeAndFlags.get(0).equals(PAIR_FLAG);
```

Comment:
Would it be better to replace the '0' here to a meaningful name to help the reader improve their understanding of the code?